src/cdk/stepper/stepper.ts

```diff
@@ -250,6 +250,11 @@ export class CdkStepper implements AfterViewInit, OnDestroy {
   /** The list of step components that the stepper is holding. */
   @ContentChildren(CdkStep) _steps: QueryList<CdkStep>;
 
+  /** The list of step components that the stepper is holding. */
+  get steps():  QueryList<CdkStep> {
```

Comment:
can you also update references to use `steps` instead of `_steps` so we can make it private in the future

Reply:
Good point. Done.

Reply:
It is also referenced in:
- src/lib/stepper/stepper.spec.ts
- src/lib/stepper/stepper-horizontal.html
- src/lib/stepper/stepper-vertical.html

Reply:
Oops, updated the references also in this files